lib/v2/screens/receive_screens/receive_enter_data/interactor/viewmodels/receive_enter_data_state.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_state.dart';
+import 'package:seeds/v2/datasource/remote/model/balance_model.dart';
+import 'package:seeds/v2/domain-shared/page_command.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+class NavigateToReceiveDetails extends PageCommand {}
+
+class ShowTransactionFailSnackBar extends PageCommand {}
```

Comment:
Move commands to their own file pls. 


`ShowTransactionFailSnackBar` should be `ShowTransactionFailMessage`

Something to help you name commands in general, The command tells the view `WHAT` to do, not `HOW` to do it. 
Its up to the view to decide if it wants to use a snakckbar, or a dialog or w.e

Reply:
If it is very necessary to put two empty classes in another file? I mean they also say that they are closely related to the state.

Reply:
It might seems overkill to move these to their own file. 
After all they are just empty classes. 

But what if they are not so trivial, we already have some page commands that have 5+ parameters and multiple page commands for one page. 

In that case, it makes sense to move to a different file to keep the state cleaner right? 
Then for the sake of consistency, they should be on a different file, even if they are small empty classes. 

Our models are closely related to the state as well, but we dont define those in the same state file. 


![image](https://user-images.githubusercontent.com/8041901/120578285-2d51d280-c3eb-11eb-9bce-6986844080a8.png)